src/MediaWiki/Page/PropertyPage.php

```diff
@@ -136,8 +136,12 @@ protected function getRedirectTargetURL() {
 
 		$label = $this->getTitle()->getText();
 
+		if ( ( $key = PropertyRegistry::getInstance()->findPropertyIdByLabel( $label ) ) === false ) {
```

Comment:
Having a side effect (variable assignment) inside of an if predicate is unexpected. I recommend against doing this for readability/understandably reasons.

This is also against the MW coding conventions if you care about those.